src/main/java/pro/sky2/Spring/EmployeeService.java

```diff
@@ -1,3 +1,53 @@
 package pro.sky2.Spring;
+
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+
+@Service
 public class EmployeeService {
+    private final int MaxEmploees = 10;
```

Comment:
```suggestion
    private final int maxEmploees = 10;
```
названия полей пишем с маленькой буквы